pkg/gadgets/trace/exec/tracer/tracer_test.go

```diff
@@ -0,0 +1,219 @@
+//go:build linux
+// +build linux
+
+// Copyright 2022 The Inspektor Gadget authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package tracer
+
+import (
+	"os/exec"
+	"testing"
+
+	"github.com/kinvolk/inspektor-gadget/pkg/gadgets/trace/exec/types"
+	utilstest "github.com/kinvolk/inspektor-gadget/pkg/gadgets/utils/test"
+	eventtypes "github.com/kinvolk/inspektor-gadget/pkg/types"
+	"golang.org/x/sys/unix"
+)
+
+func TestCreateTracer(t *testing.T) {
```

Comment:
This test is named for creating a tracer, but the last assert is about Stop() function being idempotent. Perhaps this should be split into 2 separate tests to communicate the intent better? Or perhaps function can be renamed to something like `Test_Stopping_tracer_is_idempotent`?